### PR TITLE
add meal metric units

### PR DESCRIPTION
**Pull request by alexandrter**

### CODE REVIEW

File: src/main/java/com/murano/service/MenuServiceImp.java [hunk 1]
```diff
@@ -58,5 +62,21 @@ public Menu getMenu(UUID restaurantUuid, String language) {
...
+        addIfBothNotEmpty(mealMetricUnits, mealEntity.getWeight1(), mealEntity.getUnit1());
+        addIfBothNotEmpty(mealMetricUnits, mealEntity.getWeight2(), mealEntity.getUnit2());
+        addIfBothNotEmpty(mealMetricUnits, mealEntity.getWeight3(), mealEntity.getUnit3());
+        addIfBothNotEmpty(mealMetricUnits, mealEntity.getWeight4(), mealEntity.getUnit4());
+        addIfBothNotEmpty(mealMetricUnits, mealEntity.getWeight5(), mealEntity.getUnit5());
+        return mealMetricUnits;
+    }
+
+    private void addIfBothNotEmpty(ArrayList<MealMetricUnit> mealMetricUnits, BigDecimal weight, MealEntity.Unit unit) {
+        if (weight != null && unit != null) {
```
Comment (donesvad) [inline src/main/java/com/murano/service/MenuServiceImp.java]: imho, it's not good just to skip if smth is null
can we throw an assertion?

File: src/main/java/com/murano/repository/entity/MealEntity.java [hunk 2]
```diff
@@ -25,6 +27,10 @@
 @NoArgsConstructor
 public class MealEntity {
 
+    public enum Unit {
+        MG, G, KG, ML, L, PI,
```
Comment (donesvad) [inline src/main/java/com/murano/repository/entity/MealEntity.java]: do we have a reason to keep it inside MealEntity class?
why not to keep the enum as a separate enum file?

File: src/main/java/com/murano/service/MenuServiceImp.java (hunk 1 above)
Comment (alexandrter) [reply src/main/java/com/murano/service/MenuServiceImp.java]: added logging as discussed

File: src/main/java/com/murano/repository/entity/MealEntity.java (hunk 2 above)
Comment (alexandrter) [reply src/main/java/com/murano/repository/entity/MealEntity.java]: done, decided to always create separate files for enums

**Review by donesvad — APPROVED** [review]

**Pull request merged**